wlog/svclog/svc1log/svc1logtests/tests.go

```diff
@@ -310,7 +311,61 @@ something/something:123`,
 				}),
 			}),
 		},
+		{
```

Comment:
I removed the duplicate safe param test because we are considering this a user error. This is the policy decided for Java. If a user adds duplicate safe params then their line will eventually not be parsed or result in a performance hit when falling back. Zerolog is difficult to enforce this and probably not worth it since you have to reverse all parameters being applied to it to catch the few bad actors.